### PR TITLE
feat: made the Signup Page

### LINKED ISSUE #26: [Feature Request]: SignUp Page
### Is your feature request related to a problem? Please describe.

..

### Describe the solution you'd like

..

### Describe alternatives you've considered

_No response_

### Developer Help

_No response_

### PR DESCRIPTION
**Pull request by shramanpaul**
fixes #26 
Made the signup page with all the required fields.

### CODE REVIEW

File: src/pages/LandingPage/index.jsx [hunk 1]
```diff
@@ -2,7 +2,6 @@ import NavBar from '../../components/NavBar';
 import HeroSection from '../../components/HeroSection';
 import Content from './content';
 import Footer from '../../components/Footer';
-
```
Comment (sambhavgupta0705) [inline src/pages/LandingPage/index.jsx:5]: Add a line here for better spacing

File: src/pages/signup/contentSignup.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          </div>
+          {/* <div className="flex items-center mb-6">
+            <div className="flex-1 text-sm text-gray-600">
+              <a href="#" className="text-blue-500 hover:underline">Forgot Password?</a>
+            </div>
+            <div className="checkb1">
+              <input type="checkbox" id="remember" className="form-checkbox" />
+              <label htmlFor="remember" className="ml-2 text-sm text-gray-600">Remember Me</label>
+            </div>
+          </div> */}
```
Comment (sambhavgupta0705) [inline src/pages/signup/contentSignup.jsx:52]: remove all the comments

**Review by sambhavgupta0705 — CHANGES_REQUESTED** [review]
Added some comments

File: src/pages/LandingPage/index.jsx (hunk 1 above)
Comment (shramanpaul) [reply src/pages/LandingPage/index.jsx]: done @sambhavgupta0705

File: src/pages/signup/contentSignup.jsx (hunk 2 above)
Comment (shramanpaul) [reply src/pages/signup/contentSignup.jsx]: done @sambhavgupta0705

**Comment by sambhavgupta0705** [comment]
@shramanpaul just follow the signup convention everywhere.
At some instances you have mentioned `signin` .
please fix it

**Comment by sambhavgupta0705** [comment]
@shramanpaul just follow the signup convention everywhere.
At some instances you have mentioned `signin` .
please fix it

**Review by sambhavgupta0705 — APPROVED** [review]

**Pull request merged**